slackbridge/bots.py

```diff
@@ -149,6 +151,30 @@ def nickserv_auth(self):
                 self.target_group_nick,
                 self.nickserv_password,
             ))
+    def privmsg(self, user, channel, message):
+        """
+        Set to handle if channel is own name (private chat)
+        """
+
+        nick = utils.nick_from_irc_user(user)
+        
+        if channel == self.nickname:
+            if not self.im_id:
+                self.im_id = self.sc.api_call(
+                        'im.open',
+                        user=self.user_id,
+                        return_im=True
+                        )['channel']['id']
+
+            log.msg(self.sc.api_call(
+                'chat.postMessage',
+                channel=self.im_id,
+                text=utils.format_slack_message(nick + ": " + message, IRCBot.users),
```

Comment:
I think the `pre-commit` hooks are complaining about this (the double-quoted string when it could be single quoted). If you run `make test` locally, then it'll run the `pre-commit` hooks over your changes to do linting, and if you run `make install-hooks`, it'll install the hooks into your `.git` directory so that when you commit it'll automatically run the hooks, which is pretty nice :)